packages/docusaurus-theme-classic/src/theme/MDXComponents/index.tsx

```diff
@@ -37,8 +37,22 @@ const MDXComponents: MDXComponentsObject = {
     return <Head {...props}>{unwrappedChildren}</Head>;
   },
   code: (props) => {
+    const inlineElements = [
+      'a',
+      'b',
+      'big',
+      'i',
+      'span',
+      'em',
+      'strong',
+      'sup',
+      'sub',
+      'small',
+    ];
```

Comment:
Maintaining this list is not ideal, but I doubt if it happens a lot anyways. We haven't heard anyone complaining so far